packages/module/rollup.config.js

```diff
@@ -106,4 +110,58 @@ export const cssVendor = {
   ],
 };
 
-export default [cjsBuild, esBuild, esBuildWithDeps, cssVendor];
+// wraps the quick starts styles within .pfext-quick-start__base
+// quickstarts-standalone.scss is used in conjunction with patternfly-nested.scss,
+// for consumers that have a different major version of PatternFly
+export const quickStartsStyles = {
+  input: 'src/styles/quickstarts-standalone-entry.ts',
+  output: false,
+  plugins: [
+    scss({
+      output: 'dist/quickstarts-standalone.css',
+      ...commonScssOptions,
+    }),
+  ],
+};
+
+// wraps PatternFly styles within .pfext-quick-start__base
+// useful for consumers that use a different major version of PatternFly
+// which might be incompatible with the PatternFly styles we depend upon
+export const pfStyles = {
+  input: 'src/styles/patternfly-nested-entry.ts',
+  output: false,
+  plugins: [
+    scss({
+      output: 'dist/patternfly-nested.css',
+      ...commonScssOptions,
+      // processor: css => {
+      //   return css.replace(':root', '.pfext-quick-start__base');
+      // },
```

Comment:
This class wrapping happens in the `.scss` file, any reason to leave the comment here?

Reply:
I'll remove the commented out stuff